src/main.cpp

```diff
@@ -5214,6 +5214,14 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             return true;
         }
 
+        // Only send one GetAddr response per connection to reduce resource waste
+        //  and discourage addr stamping of INV announcements.
+        if (pfrom->fSentAddr) {
+            LogPrint("net", "Ignoring repeated \"getaddr\". peer=%d\n", pfrom->id);
```

Comment:
Cosmetic nits: The dot before peer looks ugly, but the same style is used above (`getaddr` from outbound connection). The space in the comment in the second line, before the word "and" in unintentional?